### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by summyalena**
### I finished the capstone for the react module, I ensured I followed through with the guidelines listed out for the project. 

- I ensured that when the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.

- I ensured that when the user clicks (or taps) on a category item, the application navigates to the details page.

- I ensured that In the details page, the webapp retrieves data from the API to show detailed data in the category.

- I ensured that when the user clicks on the "Back" button (<), the user navigates to the home page.

- I created unit tests for pure functions in your Redux code.
- I created integration tests for your application using the React Testing Library.
- I ensured the data retrieved from the API should be stored in the Redux store.
- I ensured I could filter the data that I retrieve from the API using a Filter stateless component.
- I made sure every page (the main page and the pages for each item) should have a unique route within the SPA.
- I made sure the project be deployed and accessible online (add a link to the online version of your app to your README file)

 **_Attached here is the link to the live version https://covidcheckssplend.netlify.app/_** 

_**Also attached here is the link to my presentation video: https://www.loom.com/share/5f7614fab40d47068112291e4fa3216f**_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+-This project was inspired by [Microverse](https://www.microverse.org)
```
Comment (CollinsTatang) [inline README.md]: - [ ] The README "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/components/CovidHome/Home.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        <h3> A Statistics of covid cases in the world</h3>
+      </div>
+      <div className={styles.picture}>
+        <img src={home} className={styles.img} alt="covid" />
+      </div>
+    </div>
+  );
+}
+
+export default Home;
```
Comment (CollinsTatang) [inline src/components/CovidHome/Home.js]: - [ ] Please the Home page has a list of categories that could be filtered using given parameters, e.g. category name.
A Search Input and clickable search button  to filter by name

- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @summyalena, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.

- [ ] Kindly do a new video [presentation](https://www.loom.com/share/aa1a3d22e9964662b34fba90d209ecb5) for your project again and highlight a piece of the code you are proud of making.

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/CovidHome/Home.js (hunk 2 above)
Comment (summyalena) [reply src/components/CovidHome/Home.js]: Hello @CollinsTatang, 

Thank you for the review but the search input feature Is not part of the curriculum requirements. Please review the requirements again. 

Thank you.

Comment (CollinsTatang) [reply src/components/CovidHome/Home.js]: Hi @summyalena 
Thank you for your kind replied 
Please we have it under your Home page as per [Project Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)
search2

File: src/components/HomePage/Category.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import { actions } from '../../redux/covid/covid';
+import IMAGES from '../../images/images';
+import loader from '../../images/1492.gif';
+
+function CovidCase() {
+  const dispatch = useDispatch();
+  const loading = useSelector((state) => state.covidCases.loading);
+  const handleCovidClick = (name) => {
+    dispatch(actions.filterCases(name));
+  };
```
Comment (DammyShittu) [inline src/components/HomePage/Category.js]: - [ ] On the homepage you are yet to implement the filtering (or search) functionality as requested by the previous reviewer. Kindly implement that as well.

  search

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @summyalena,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @summyalena, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**